src/atsu/src/messages/DclMessage.ts

```diff
@@ -31,9 +31,9 @@ export class DclMessage extends CpdlcMessage {
     public serialize(format: AtsuMessageSerializationFormat) {
         let dclMessage = '';
         if (format === AtsuMessageSerializationFormat.Network) {
-            dclMessage = 'REQUEST PREDEP CLEARANCE\n';
-            dclMessage += `${this.Callsign} ${this.AcType} TO ${this.Destination}\n`;
-            dclMessage += `AT ${this.Origin}${this.Gate !== '' ? ` STAND ${this.Gate}` : ''}\n`;
+            dclMessage = 'REQUEST PREDEP CLEARANCE \n';
+            dclMessage += `${this.Callsign} ${this.AcType} TO ${this.Destination} \n`;
+            dclMessage += `AT ${this.Origin}${this.Gate !== '' ? ` STAND ${this.Gate}` : ''} \n`;
```

Comment:
Just because the issue what that filed here I'm curious - I assume the new lines are used by some kind of controller client, right? Otherwise we wouldn't have them?

Reply:
There is no real standard about it. I checked in the Hoppie-logs what other A/Cs are sending and the way as this PR will do it.
I know that most modern controller software has no problem with it, but I guess that vStrip is a bit picky here.

The newlines are mainly for the readability of the controller. It is sometimes necessary to read the request manually.
I guess this was the initial idea (years ago) behind the newlines

Reply:
Alright, great.